### PR TITLE
Fix localization issues

### PR DESCRIPTION
**Pull request by rabelux**

### CODE REVIEW

File: Contents/Code/__init__.py [hunk 1]
```diff
@@ -190,13 +190,23 @@ def search(self, results, media, lang, manual):
...
+        localized_sep = by_lang_dict.get(lang, "by")
+        log.debug("Localized separator between title and artist: " + localized_sep)
 
         # Output the final results.
         log.separator(log_level="debug")
         log.debug('Final result:')
         for i, r in enumerate(info):
-            description = '\"%s\" by %s [%s]' % (
-                r['title'], r['artist'], r['year']
+            description = '[%s] \"%s\" %s %s' % (
```
Comment (djdembeck) [inline Contents/Code/__init__.py:208]: @seanap what do you think about this style change?

Comment (djdembeck) [reply Contents/Code/__init__.py:208]: It looks like the year starts to be truncated at 81 characters. Alternatives would be to:

1. Truncate the search result text ourselves.
2. Only show year first if the `description` is 81 characters or more.

The issue is that with the year in front, the author is now more than likely going to be truncated. So to me, it seems this is a game of picking which data is most important.

Comment (rabelux) [reply Contents/Code/__init__.py:208]: For me it starts truncating at 60 characters.
grafik
Is it possible to add a column like there is for the score? That way the year can have a fixed spot at the end and doesn't bounce back an forth.

Comment (rabelux) [reply Contents/Code/__init__.py:208]: Or is it possible to change font styles? Then we could leave the year at the front with a greyish color and leave out the brackets (saving 2 chars)

Comment (rabelux) [reply Contents/Code/__init__.py:208]: Having a max length for author and/or title does sound like a good idea though. Only question is where to draw the line.

**Comment by rabelux** [comment]
Let's split the fix for localization issues and the discussion about year positioning.

**Review by djdembeck — APPROVED** [review]

**Comment by djdembeck** [comment]
After thinking it over, I think year being in front makes sense, even if it's not the norm of other agents.

**Pull request merged**